### ISSUE TITLE
Need custom pages for each new tag used

### ISSUE THREAD

**Issue by mayarichman** [youngist/clean-blog#6]
#### **Priorities**

Choosing which posts from [youngist.org](youngist.org) get migrated, making a list of **tags** for the **migrated posts**, and a list of common tags **all posts** in general is the next step.

@Acker1021 @awstevens  @catachresis  @qaween If you haven't filled it out, can everyone choose which tags they find important in this [google doc](https://docs.google.com/document/d/13QZiWFwTq6Qygt9Mo1Tm1a9KB3hEM1tOj4586oHIAI0/edit).

@awstevens Can you flag which tumblr posts are to be migrated and newer ones as well when you have the time?

---
## Technical Information

I have discovered that the functionality for pages for tags is not part of the core of Jekyll and so therefore is not supported on Github pages _yet_. In terms of custom pages for each tag, i.e. `youngist.org/tag/women` we have more limited options:
1. We could make separate html pages for each tag that aggregate posts with that tag.

`_posts/tags/2013-07-10-women.md` would just have front matter that specified the tag was women and an include that displayed the tag page as designed by @Acker1021 

``` yaml
----
layout: tag
tag: women
----
{% include tag_layout.html %}

```

`tag_layout.html`

``` html

<h3 class='tag-title'>{{page.tag}}</h3>
{% for post in site.posts %}
{% if post.tag == {{page.tag}} %}
//show article if article is tagged with this {{page.tag}} from the post front matter
{% endfor %}

```

But this would mean that every time we wanted to add a new tag, someone would have to make a new post in the `_posts/tags/` directory, similar to the one above.
1. A second option would be to choose a few tags for now that repeat and so we wouldn't need to keep adding new tags. While we wait to see if this gets improved (which is happening right now according to this issue mojombo/jekyll#870
2. The problem is where we host the site, Github Pages, not the platform in general. We could look into hosting elsewhere, such as S3, where we could use plugins [Options](http://jekyllrb.com/docs/deployment-methods/). However this may be more technically complex for me and those editing.
3. If the site is built on my computer and the changes pushed to Github then that works too, however, every time a post is made I would need to do this and this seems to be a bad workflow.

---
## Solution

I think the best option for now is the custom pages with common tags and to wait to hear back about the upgrade. According to [mattr-](https://github.com/mattr-): 

If we had a million tags we wanted to use, like on tumblr this would be a horrible decision, however I think we wanted simplicity to start anyways and it wouldn't be much work for me once I knew which tags we wanted to use.

> We're actually waiting on @ixti to submit what he thinks is the real
> pull request for this.

**Comment by awstevens** [comment]
I don't have permission to access the Google Doc

**Comment by mayarichman** [comment]
@awstevens oh ok, no prob, started a wiki [here](https://github.com/youngist/clean-blog/wiki/Migrating-Posts). You should have rights to edit it. :+1:

**Comment by mayarichman** [comment]
I think not making custom pages with pictures for every tag with save everyone time and stress. Tag pages are generated automatically when you use them, however they just don't have cool pictures associated. These pictures however would be a huge discussion. We can think about this more, but right now I am using youngist orange in place of a large header image. We could also use a photo that would work for all youngist things to appear on that page.
